src/style.css

```diff
@@ -2,19 +2,21 @@ body {
   margin: 0;
 }
 
-div {
+.block-red,
+.block-green,
+.block-blue {
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.